src/demo/resources/demo/db/postgres.types.ben.xml

```diff
@@ -25,8 +25,7 @@
         "text"      TEXT NOT NULL,
         "binary"    BYTEA NOT NULL,
         "json"      JSON NOT NULL,
-        "jsonb"     JSONB NOT NULL,
-        "genom"     GEOMETRY NOT NULL
```

Comment:
??? why did you remove genom

Reply:
I remove it for my local test but not forgot to add again. I test it because sometime GEOMETRY type will cause ERROR:  type "geometry" does not exist. That happen when the user postgers don't have postgis EXTENSION (can be fix by code: CREATE EXTENSION postgis;)

Reply:
yes we need to fix it ...

`postgis EXTENSION (can be fix by code: CREATE EXTENSION postgis;)`